app/src/store/wallets/actions.js

```diff
@@ -349,8 +349,8 @@ export const actions = {
       }
 
 
-      if (responseBalances.data.result[0]) {
-        available = parseInt(responseBalances.data.result[0].amount)
+      if (responseBalances.data.result.length) {
+        available = parseInt(responseBalances.data.result.find(balance => balance.denom === 'uxki').amount);
```

Comment:
```suggestion
        available = parseInt(responseBalances.data.result.find(balance => balance.denom === config.kichain.udenom).amount);
```

Reply:
And you'll need to add the correct import

Reply:
I didn't know the constants for the denom, and I didn't look. thanks to you, it's cleaner! :p

Reply:
Don't forget to add the import for the config

Reply:
done ;-)